lib/components/Form/AutoSuggest/AutoSuggest.tsx

```diff
@@ -12,21 +12,25 @@ interface Option {
 export interface AutoSuggestProps extends FieldComponentProps {
   disabled?: boolean
   options?: Option[]
+  onChange?: (option: string | null | undefined) => void
 }
 
 export const AutoSuggest = ({ ...props }: AutoSuggestProps) => {
   const { name, options, disabled = false, ...rest } = props
   const [field, meta] = useField(name)
   const { setFieldValue } = useFormikContext<unknown>()
+
   const errorClass = useErrorClass(name)
 
   const [selectedOption, setSelectedOption] = useState<Option | null>(null)
 
-  useEffect(() => {
-    const initialOption = options?.find((option) => option.value === field.value)
+  console.log(field.value)
 
+  useEffect(() => {
+    const initialOption = options?.find((option) => option.value === meta?.initialValue)
     setSelectedOption(initialOption || null)
-  }, [field.value, options])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Is there a way to do this without eslint disable? Code likes this makes me nervous for reloads.

Reply:
i am fixing this , trying to change the way it is functioning